steamcmd_servers/suicidesurvival/egg-suicide-survival.json

```diff
@@ -0,0 +1,95 @@
+            "container": "debian:buster-slim",
+            "entrypoint": "bash"
+        }
+    },
+    "variables": [
+        {
+            "name": "APP ID",
+            "description": "The ID corresponding to the game to download.",
+            "env_variable": "APPID",
+            "default_value": "310",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "required|numeric|digits_between:1,6"
+        },
+        {
+            "name": "Maximum Players",
+            "description": "",
+            "env_variable": "MAX_PLAYERS",
+            "default_value": "12",
+            "user_viewable": true,
+            "user_editable": true,
```

Comment:
You should mark this as read only

Reply:
Shouldn't users be able to change how many people can join the server?